www/grpc/blockchain.go

```diff
@@ -219,13 +219,23 @@ func (s *blockchainServer) GetValidator(_ context.Context,
 func (s *blockchainServer) GetValidators(_ context.Context,
```

Comment:
I feel we don't need this function. This is replaced with `GetValidatorAddresses`. Isn't it?

Reply:
This method return list of commitee validators, if we don't need to this method we can remove in new issue.